uavcan/diagnostic/65520.Record.1.0.uavcan

```diff
@@ -0,0 +1,17 @@
+#
+# Generic human-readable text message for logging and displaying purposes.
+# Generally it should be published at the lowest priority level.
+#
+
+# Optional timestamp; zero if undefined.
+uavcan.time.Point.1 timestamp
```

Comment:
The changes in #50 makes this more sensible as:
 - We have an explicit timeframe meaning we can plot the time and date in a human friendly way
 - We can send a "Time since boot" `SystemTime` as a fallback when no time sync exists, allowing us to remove the notion of a special for zero.